base/strings/basic.jl

```diff
@@ -56,18 +56,19 @@ sizeof(s::AbstractString) = error("type $(typeof(s)) has no canonical binary rep
 eltype(::Type{<:AbstractString}) = Char
 
 """
-```
-*(s::AbstractString, t::AbstractString)
-```
+    *(s::Union{Char, AbstractString}, t::Union{Char, AbstractString}...)
 
-Concatenate strings. The `*` operator is an alias to this function.
+Concatenate strings and characters. The `*` operator is an alias to this function.
```

Comment:
"[...] and characters to a ``[`String`](@ref)``." perhaps?

Also, this is the `*` function, its an alias for itself!?